velox/expression/tests/ExprTest.cpp

```diff
@@ -1072,6 +1101,23 @@ TEST_F(ExprTest, constantFolding) {
     auto typedExpr = parseExpression("codepoint('abcdef')");
     EXPECT_NO_THROW(exec::ExprSet exprSet({typedExpr}, execCtx_.get(), true));
   }
+
+  // Test constant-folding when the expression has default null behavior and at
+  // least one input is null.
+  VELOX_REGISTER_VECTOR_FUNCTION(udf_default_null, "default_null");
+  typedExpr = parseExpression("sign(default_null(null, 0.1))");
```

Comment:
I wonder if this expression would be constant folded even without the changes in this PR. I think we need to use an expression where not all inputs are constants, e.g. sign(floor(null) + c0)

Reply:
You're right. I fixed this expression in the new version.